### ISSUE TITLE
Embedded lyrics missing when using "shuffle all"

### ISSUE THREAD

**Issue by jixunmoe** [jeffvli/feishin#175]
## Expected Behavior

Lyrics been fetched and shown.

## Current Behavior

Lyrics missing when using "shuffle all".

## Steps to Reproduce

1. Create a navidrome instance where all songs contained LRC lyrics embedded to the media and indexed.
2. "Shuffle all" from the interface.
3. No lyrics available when switched to the Lyrics tab.

## Possible Solution (Not obligatory)

Workaround: play from tracks section and manually shuffle from there.

## Context

The old client (sonixd) is able to display the lyrics text content as it is.

When requesting the random api (`/rest/getRandomSongs.view`), the response does not include lyrics:

```json
{
  "subsonic-response": {
    "status": "ok",
    "version": "1.16.1",
    "type": "navidrome",
    "serverVersion": "0.49.3 (8b93962f)",
    "randomSongs": {
      "song": [
        {
          "id": "5bbb499925c655b4147298e40785dc93",
          "parent": "61c01b1ddfe2f4872fc39888a1793e0c",
          "isDir": false,
          "title": "玫瑰少年",
          "album": "UGLY BEAUTY",
          "artist": "蔡依林",
          "track": 2,
          "year": 2018,
          "coverArt": "mf-5bbb499925c655b4147298e40785dc93_64921e97",
          "size": 4879472,
          "contentType": "audio/mp4",
          "suffix": "m4a",
          "duration": 191,
          "bitRate": 200,
          "path": "蔡依林/UGLY BEAUTY/02 - 玫瑰少年.m4a",
          "playCount": 3,
          "played": "2023-07-04T22:28:05Z",
          "discNumber": 1,
          "created": "2023-06-20T22:06:02.855677599Z",
          "albumId": "61c01b1ddfe2f4872fc39888a1793e0c",
          "artistId": "060545170130c81c5505fda49b58d018",
          "type": "music",
          "isVideo": false
        }
      ]
    }
  }
}
```

When playing directly from the tracks interface, it makes a request to song `/api/song` which had `lyrics` populated.

The same issue can also be observed (not always!) from the album or search page, as those filtering endpoints does not provide `lyrics` field neither.

## Your Environment

-   Application version (e.g. v0.1.0) : v0.2.0 (https://github.com/jeffvli/feishin/releases/tag/v0.2.0)
-   Operating System and version (e.g. Windows 10) : Windows 11
-   Server and version (e.g. Navidrome v0.48.0) : [Navidrome v0.49.3](https://github.com/navidrome/navidrome/releases/tag/v0.49.3) (8b93962f)
-   Node version (if developing locally) : N/A, using the Windows installer.

**Comment by kgarner7** [comment]
When you upgrade to Navidrome 0.51 (or later releases), this will be resolved by #484. This issue (as you noted) comes up because the getRandomSongs doesn't return a full Navidrome track object.

**Comment by kgarner7** [comment]
I'm going to close this, as it is resolved using Navidrome 0.51 and development Feishin https://github.com/jeffvli/feishin/commit/83d5fee4426ae8ac41662debadf613ca21c0e045 (no longer relying on the `song` `lyrics` property)
